### PR TITLE
Specify what to do when a TA gets compromised

### LINKED ISSUE #20: Explain what happens when a TA's key gets compromised
Basically the same as with sources, so we can simply link to the paragraph in `sources.md`, but this needs to be spec'd.

### PR DESCRIPTION
**Pull request by vabd**
Resolves #20

### CODE REVIEW

File: content/trust-management/trust-authority.md [hunk 1]
```diff
@@ -302,6 +302,22 @@ not** include the compromised key.
 }
 ```
 
+## Compromission of cryptographic private keys
```
Comment (GordonF42) [inline content/trust-management/trust-authority.md]: Compromission => getting compromised

File: content/trust-management/trust-authority.md [hunk 2]
```diff
@@ -302,6 +302,22 @@ not** include the compromised key.
...
+
+In the event of a trust authority having one or more of its cryptographic
+private keys compromised, it **must** get in touch with every trust authority
+certifying it as trustworthy and follow the same instructions as if it was a
+source ([described here]({{<relref
+"/information-distribution/source#compromission-of-cryptographic-private-keys">}})).
```
Comment (GordonF42) [inline content/trust-management/trust-authority.md]: this link is probably wrong since the target title has changed

**Review by GordonF42 — DISMISSED** [review]
some issues around the compromission => getting compromised changes

otherwise LGTM :)

File: content/trust-management/trust-authority.md (hunk 1 above)
Comment (vabd) [reply content/trust-management/trust-authority.md]: Gah, I fixed it in the wrong file...

File: content/trust-management/trust-authority.md (hunk 2 above)
Comment (vabd) [reply content/trust-management/trust-authority.md]: Good point.

**Review by GordonF42 — APPROVED** [review]
LGTM

**Pull request merged**